public/lib/date-service.js

```diff
@@ -136,6 +136,20 @@ angular.module('wfDateService', ['wfLocationService'])
             return this.createRange(dayStart, dayStart.clone().add(1, 'days'));
         }
 
+        /**
+         * Minus a second from a "from" date to avoid midnight dates being filtered out
```

Comment:
Could we amend/expand this comment slightly? This would be applicable in any situation where we want to filter from a specific time onward, e.g. if we had a filter for 'This afternoon' that had to include everything with a deadline from 12pm onwards. Maybe we could add a little context about how the query works and why we need this work around.

Reply:
You're right, that comment is a bit sparse. I will add some context to it.